### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by Kong12384abc** [community/community#11008] [Copilot Conversations]
[ERROR] [default] [2022-02-04T13:35:26.158Z] GitHub Copilot could not connect to server. Extension activation failed: "Timed out waiting for authentication provider to register"

(my github account has been added in whitelist)

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by Kong12384abc** [reply]
No, I disable Windows Defender's firewall 
And i dant have vpn

**Comment by msaroufim** [comment]
I'm getting the same error and it's persisting regardless of how many times I reinstall or logout. Am not behind a VPN or firewall

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
